app/Console/Commands/RefreshTesting.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use Illuminate\Support\Facades\Artisan;
+
+    Artisan::command('test:refresh', function () {
+        $this->call('migrate:fresh', [
+            '--env' => 'testing',
+        ]);
+        $this->call('db:seed', [
+            '--env' => 'testing',
+            '--class' => 'DemoSeeder',
+        ]);
+    })->describe('Resets the testing database and seeds the demo data.');
```

Comment:
I'm not sure if we should "polute" the test namespace with our own commands. Maybe create our own spacialist namespace where we can also add other commands (e.g. create users from cli, ...)